src/helix-ui/elements/HXAlertElement.js

```diff
@@ -166,26 +166,32 @@ export class HXAlertElement extends HXElement {
     }
 
     /**
-     * Programmatically dismiss the alert (removes element from the DOM).
+     * Dismiss the alert (removes element from the DOM).
      */
     dismiss () {
-        this.remove();
+        if (this.$emit('dismiss')) {
+            // only if event was not canceled by consumer
+            this.remove();
+        }
+    }
+
+    /**
+     * Simulate a mouse click on the CTA button.
+     */
+    submit () {
+        this.$emit('submit');
     }
 
     /** @private */
     _onDismiss (evt) {
         evt.preventDefault();
-
-        if (this.$emit('dismiss')) {
-            // only if event was not canceled by consumer
-            this.dismiss();
-        }
+        this.dismiss();
```

Comment:
How does moving this into its own public function make it so that a user's click will interact with light dom?

Reply:
So users are still clicking shadow dom to close the modal?

Reply:
yes, the `dismiss()` method is just exposing the same functionality via JavaScript